content/en/docs/tasks/administer-cluster/topology-manager.md

```diff
@@ -223,12 +223,11 @@ You can toggle groups of options on and off based upon their maturity level usin
 
 You will still have to enable each option using the `TopologyManagerPolicyOptions` kubelet option.
 
-### `prefer-closest-numa-nodes` (beta) {#policy-option-prefer-closest-numa-nodes}
+### `prefer-closest-numa-nodes` (stable) {#policy-option-prefer-closest-numa-nodes}
 
-The `prefer-closest-numa-nodes` option is beta since Kubernetes 1.28. In Kubernetes {{< skew currentVersion >}}
-this policy option is visible by default provided that the `TopologyManagerPolicyOptions` and
-`TopologyManagerPolicyBetaOptions` [feature gates](/docs/reference/command-line-tools-reference/feature-gates/)
-are enabled.
+The `prefer-closest-numa-nodes` option is GA since Kubernetes 1.32. In Kubernetes {{< skew currentVersion >}}
+this policy option is visible by default provided that the `TopologyManagerPolicyOptions`
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/) is enabled.
```

Comment:
Em ... isn't this feature "locked-to-true"?
In other words, is it possible to disable it after GA?

Reply:
Ha this one is kinda complex. So for GA the behavior stays the same as for beta. The `prerfer-closest-numa-nodes-nodes` is disabled by default and has to be enabled by setting `prefer-closest-numa-nodes=true` in kubelet config. But if you disable `TopologyManagerPolicyOptions` feature gate then you won't be even able to set this parameter. This feature follows the same pattern as `CPUManagerPolicyOptions` which are already stable 😄  

So answering your question:
- no this feature is not locked-to-true, by default it's not enabled
- yes, you can disable/enable it by passing special params to kubelet